platform_umbrella/apps/home_base_web/config/runtime.exs

```diff
@@ -37,6 +37,11 @@ config :common_core, CommonCore.JWK,
   sign_key: :environment,
   verify_keys: [:home_a_pub, :home_b_pub]
 
+config :home_base, HomeBase.Mailer,
+  adapter: Swoosh.Adapters.Postmark,
+  # Fall back to the black hole postmark server if no key is provided
+  api_key: System.get_env("POSTMARK_KEY") || "d382f183-4b4b-417f-82db-3f9635a05c8b"
```

Comment:
Isn't the blackhole server meant to test bounces? Might be better to use `POSTMARK_API_TEST`, which returns valid API responses but doesn't actually send anything (https://postmarkapp.com/blog/handling-email-in-your-test-environment)

Reply:
This is a sandbox server that I created in postmark. It will return success in the Api, like their test env, the only difference is that we can see the sent emails in our account. I invited you and @JTarasovic to the postmark account.